app/helpers/application_helper.rb

```diff
@@ -37,4 +37,26 @@ def render_flash_stream
     turbo_stream.update "flash", partial: "layouts/flash"
   end
 
+  def show_department_fullname(id)
+    Department.find(id).display_fullname
+  end
+
+  def show_role_title(id)
+    Role.find(id).show_role_title
+  end
+
+  def department_admins( department )
+    #collection of a department's admins
+    deptadmin_permissions = Permission.where(role_id: department_admin_role_object, department_id: department)
+    deptadmin_accesses = Access.where(permission_id: deptadmin_permissions).pluck(:uniqname)
+    deptadmin_object_collection = User.where(uniqname: deptadmin_accesses)
+  end
+
+  def department_recorders( department = nil )
+    #collection of a department's recorders
+    recorder_permissions = Permission.where(role_id: recorder_role_object, department_id: department)
```

Comment:
Where is _recorder_role_object_ defined ?

Reply:
This was removed and the helper here is a vestige that also is now removed